tests/melt_transport_convergence_simple.cc

```diff
@@ -0,0 +1,298 @@
+#include <aspect/material_model/interface.h>
+#include <aspect/velocity_boundary_conditions/interface.h>
+#include <aspect/fluid_pressure_boundary_conditions/interface.h>
+#include <aspect/simulator_access.h>
+#include <aspect/global.h>
+#include <aspect/melt.h>
+
+#include <deal.II/dofs/dof_tools.h>
+#include <deal.II/numerics/data_out.h>
+#include <deal.II/base/quadrature_lib.h>
+#include <deal.II/base/function_lib.h>
+#include <deal.II/numerics/error_estimator.h>
+#include <deal.II/numerics/vector_tools.h>
+
+const double c = 1.0;
+const double pi = 3.14159265359;
```

Comment:
pi is also declared in numbers::PI, and probably more accurate (although I do not know if you need it accurate here)